### PR TITLE
retain filters on the proposal route

### PR DESCRIPTION
**Pull request by hgwood**
This is step 2 from issue #178.

### CODE REVIEW

File: src/store/listeners/reactions/proposals.js [hunk 1]
```diff
@@ -39,7 +39,13 @@ export const selectProposal = reaction(async (action, store, { router }) => {
   const proposalIndex = proposalKeys.indexOf(proposalId)
   if (proposalIndex !== -1) {
     store.ui.organizer.proposal.set({ proposalIndex })
-    router.push(`/organizer/event/${eventId}/proposal/${proposalId}`)
+    const filters = store.ui.organizer.proposals.get()
+    router.push({
+      pathname: `/organizer/event/${eventId}/proposal/${proposalId}`,
+      query: {
```
Comment (fabienjuif) [inline src/store/listeners/reactions/proposals.js]: Why do you clone the object here ?
You don't trust the router API ?

```js
router.push({
  pathname: `/organizer/event/${eventId}/proposal/${proposalId}`,
  query: filters,
})
```

Comment (hgwood) [reply src/store/listeners/reactions/proposals.js]: Nope, just brain farted. Thanks.

File: src/store/listeners/reactions/proposals.js [hunk 2]
```diff
@@ -39,7 +39,11 @@ export const selectProposal = reaction(async (action, store, { router }) => {
   const proposalIndex = proposalKeys.indexOf(proposalId)
   if (proposalIndex !== -1) {
```
Comment (bpetetot) [inline src/store/listeners/reactions/proposals.js:40]: I know that is not your code, but can you change this condition with : 
``` js
if (proposalKeys.includes(proposalIndex))
```

_es6 way of code_ :)

**Review by bpetetot — COMMENTED** [review]
👌 clear and effective. Thx

File: src/store/listeners/reactions/proposals.js (hunk 2 above)
Comment (hgwood) [reply src/store/listeners/reactions/proposals.js:40]: Well I need the index on line 42 :)

Comment (bpetetot) [reply src/store/listeners/reactions/proposals.js:40]: oh sorry, I missed that ;)

**Review by bpetetot — APPROVED** [review]

**Pull request merged**